src/EditorFeatures/Core/Commands/GoToDissassemblyCommandArgs.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+
+namespace Microsoft.CodeAnalysis.Editor.Commands
+{
+    /// <summary>
+    /// Arguments for Go To Implementation.
+    /// </summary>
+    [ExcludeFromCodeCoverage]
+    internal sealed class GoToDisassemblyCommandArgs : CommandArgs
```

Comment:
:memo: @dotnet/roslyn-ide Currently the pull request ties directly in with Go To Definition, meaning the newly defined Go To Disassembly command is not used. We need to decide whether we want to initially implement this feature using two separate commands, or from the start make this an extension of the existing Go To Definition command.

Reply:
➡️ Consolidated to using the existing command in 7129e16fba9685274d287ee9ae3c977297119561.